src/app.ts

```diff
@@ -1,7 +1,24 @@
 import "./components/maincontroller";
 
-let senders = cast.framework.CastReceiverContext.getInstance().getSenders();
-let id = senders.length !== 0 && senders[0].id ? senders[0].id : new Date().getTime();
+declare global {
+    let PRODUCTION: boolean;
+    interface Window {
+        deviceInfo: deviceInfo;
+        mediaElement: HTMLElement;
+        playlist: Array<any>;
+        currentPlaylistIndex: number;
+        repeatMode: "RepeatOne" | "RepeatAll" | "RepeatNone";
```

Comment:
It could be interesting down the line to use an enum for this.

You'd essentially be able to use `RepeatMode.RepeatOne` or similar everywhere.

Reply:
Yeah that's a good idea